programs/openbook-v2/Cargo.toml

```diff
@@ -53,3 +53,4 @@ solana-program-test = { workspace = true }
 solana-logger = { workspace = true }
 spl-token = { version = "^3.5.0", features = ["no-entrypoint"] }
 spl-associated-token-account = { workspace = true, features = ["no-entrypoint"] }
+mpl-utils = { version = "0.3.2", features = ["spl-token"] }
```

Comment:
i don't think this dependency is required, the code doesn't seem to use it

Reply:
hm, this worked around compile problem with _mpl-token-metadata-1.13.2_:

> could not find 'token' in 'mpl_utils'

does it compile for you without the dependency?

Reply:
Strange. using the patched anchor fixed it for me